### ISSUE TITLE
Newest fasmg breaks ti84pce.inc

### ISSUE THREAD

**Issue by PeterTillema** [jacobly0/fasmg-ez80#9]
When trying to compile ICE debugger with the latest fasmg and latest include files, I got this error:
```
flat assembler  version g.ibh5n
src/icedebug.asm [2] src/../include/library.inc [2] src/../include/ti84pceg.inc [52] src/../include/ti84pce.inc [3340]:
        lcdBpp1 equ 0000b | lcdTft | lcdEn
macro equ [1] macro assign [4]:
        assign name, left or right
Processed: assign lcdBpp1, 0000b or lcdTft | lcdEn
Error: illegal instruction.
```

**Comment by jacobly0** [comment]
Nothing supports the latest fasmg update yet, will fix eventually...

**Comment by jacobly0** [comment]
Fixed-ish 2cc230eacbc4f4d97a447605fd205b19faf6add2
